tests/test_all_rules.py

```diff
@@ -546,20 +546,22 @@ def test_deprecated_rules(self):
         versions = loaded_version_lock.version_lock
         deprecations = self.rules_config.deprecated_rules
         deprecated_rules = {}
-        rules_path = self.custom_dir if self.custom_dir else get_path('rules')
-        deprecated_path = Path(rules_path).joinpath("_deprecated")
+        rules_paths = RULES_CONFIG.rule_dirs
 
         misplaced_rules = []
         for r in self.all_rules:
             if "rules_building_block" in str(r.path):
                 if r.contents.metadata.maturity == "deprecated":
                     misplaced_rules.append(r)
-            elif "_deprecated" in r.path.relative_to(rules_path).parts \
-                    and r.contents.metadata.maturity != "deprecated":
-                misplaced_rules.append(r)
+            else:
+                for rules_path in rules_paths:
+                    if "_deprecated" in r.path.relative_to(rules_path).parts \
```

Comment:
is_relative_to ?

Reply:
For this one we want to leave it as it is part of the check